deploy/sa.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: default
```

Comment:
Please call it `marketplace-operator-serviceaccount.` This change should also be reflected in rbac.yaml. And please run the deployment to ensure that everything works as before.